iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -688,8 +690,8 @@ public MultilineString MULTILINE_STRING() {
         b.token(Punctuator.TRIPLE_APOSTROPHE)));
   }
 
-  public FunctionCall FUNCTION_CALL() {
-    return b.<FunctionCall>nonterminal(BicepLexicalGrammar.FUNCTION_CALL).is(
+  public Expression FUNCTION_CALL() {
```

Comment:
Okay, for now, but the logic is now entirely in the Factory. We could also move it to the grammar. However, let's do it in a refactoring.